.buildkite/src/Command/Docker/Dsl.dhall

```diff
@@ -0,0 +1,23 @@
+-- Docker plugin specific settings for commands
+--
+-- See https://github.com/buildkite-plugins/docker-buildkite-plugin for options
+-- if you'd like to extend this definition for example
+--
+-- TODO: Move volume to something in the cloud or artifacts from gcloud storage
+
+let Docker = ./Type.dhall
+
+let Config = {
+  Type = {
+    image: Text
+  },
+  default = {=}
+}
+
+let build : Config.Type -> Docker.Type = \(c: Config.Type) ->
+  Docker::{ image = c.image }
+
+in
+
+{Config = Config, build = build }
```

Comment:
What's the point of this file? Why not just use the docker schema directly from the outside? `Docker::{ image = ... }`? This seems like a noop.

Reply:
I predicting we add a record attribute for tags and use that to compute the underlying image, but yeah, let's just make this file later if we do that.